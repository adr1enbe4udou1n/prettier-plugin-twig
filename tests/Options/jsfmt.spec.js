run_spec(__dirname, ["twig"], {
    twigAlwaysBreakObjects: false,
    twigPrintWidth: 120,
    twigOutputEndblockName: true
});
