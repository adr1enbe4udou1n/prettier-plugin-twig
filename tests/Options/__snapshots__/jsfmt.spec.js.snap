// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`alwaysBreakObjects.twig - twig-verify > alwaysBreakObjects.twig 1`] = `
<section class="{{ {
    base: css.prices
} | classes }}">
</section>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<section class="{{ { base: css.prices }|classes }}"></section>

`;

exports[`endblockName.twig - twig-verify > endblockName.twig 1`] = `
{% block title %}
    {{ item.name|title }}
{% endblock %}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{% block title %}
    {{ item.name|title }}
{% endblock title %}

`;

exports[`printWidth.twig - twig-verify > printWidth.twig 1`] = `
<span attr1="one" attr2="two" attr3="three" attr4="four" attr5="five" attr6="six" attr7="seven" attr8="abcd">Text</span>

<span attr1="one" attr2="two" attr3="three" attr4="four" attr5="five" attr6="six" attr7="seven" attr8="abcde">Text</span>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<span attr1="one" attr2="two" attr3="three" attr4="four" attr5="five" attr6="six" attr7="seven" attr8="abcd">Text</span>

<span attr1="one" attr2="two" attr3="three" attr4="four" attr5="five" attr6="six" attr7="seven" attr8="abcde">
    Text
</span>

`;
